### DISCUSSION TITLE
Circular Picker

### DISCUSSION THREAD

**Discussion by JosipDomazetDev** [mchome/flutter_colorpicker#71] [Q&A] [answered]
Hi I was wondering how to implement a circular color picker like the one in the screenshot? I could not find it in the examples.

![image](https://user-images.githubusercontent.com/69600731/145731929-025696b0-ee11-4917-b1fd-6a23847be953.png)

**Comment by mchome** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/7392658/148155000-230440a4-cb91-4ac5-b960-5c3bb7f1490a.png)
![image](https://user-images.githubusercontent.com/7392658/148155037-12f9b178-0311-43ab-8f67-9a190466113f.png)
